app.rb

```diff
@@ -0,0 +1,158 @@
+require_relative './classroom'
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def run
+    prompt_user
+  end
+
+  def quit_app
+    puts 'Thank you for using this app! Now exiting...😊'
+    exit
+  end
+
+  # Person: Student /Teacher
+  def list_all_people
+    if @people.empty?
+      puts 'The people list is empty, add some people...😀'
+    else
+      puts "People's list, count(#{@people.count})🧑🏿‍🤝‍🧑🏻 :\n\n"
+      @people.each_with_index do |person, index|
+        puts "#{index + 1}) Type: #{person.type}, Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_person
+    puts 'Which do you want to create? A student (press 1) or a teacher (press 2)?'
+    choice = gets.chomp.to_i
+    case choice
+    when 1
+      register_student
+    when 2
+      register_teacher
+    end
```

Comment:
- [ ] If a user enters something else beside 1 or 2 let them know that they input a incorrect number

Reply:
Thanks @GzimAsani , I will make that fix